ruby-3.3.yaml

```diff
@@ -49,7 +49,7 @@ pipeline:
     with:
       repository: https://github.com/ruby/ruby
       tag: v${{vars.underscore-package-version}}
-      expected-commit: be1089c8ec5ba40e09b1553e36b3174bf4014d9d
+      expected-commit: ef084cc8f4958c1b6e4ead99136631bef6d8ddba
 
   - uses: patch
```

Comment:
@mamccorm 🤔 ruby 3.3.5 ships with rexml 3.3.6, I think the `patch` is not needed anymore and the reason for the failing build.

Reply:
Thanks @neumayr. Indeed, looks like it's been bumped upstream in the last few days:
 - https://github.com/ruby/ruby/blame/v3_3_5/gems/bundled_gems#L12